Error when you use ux-turbo without mercure

I got an error "[ERROR] Please run "composer require symfony/mercure". It is needed to broadcast entities. " comes from 
src/Maker/MakeEntity line 146. In fact, i'm using turbo without using mercure. 

```php
        if (
            !$input->getOption('broadcast')
            && class_exists(Broadcast::class)
            && !class_exists($this->generator->createClassNameDetails($entityClassName, 'Entity\\')->getFullName())
        ) {
            // Mercure is needed
            if (!class_exists(MercureExtension::class)) {
                throw new RuntimeCommandException('Please run "composer require symfony/mercure". It is needed to broadcast entities.');
            }

            $description = $command->getDefinition()->getOption('broadcast')->getDescription();
            $question = new ConfirmationQuestion($description, false);
            $isBroadcast = $io->askQuestion($question);

            $input->setOption('broadcast', $isBroadcast);
        }
```

The conditions tested on these lines of code are too broad. Indeed, when we use ux-turbo without using Mercure we generate the exception

--- a/src/Maker/MakeEntity.php
+++ b/src/Maker/MakeEntity.php
@@ -141,14 +141,14 @@ final class MakeEntity extends AbstractMaker implements InputAwareMakerInterface
             && class_exists(Broadcast::class)
             && !class_exists($this->generator->createClassNameDetails($entityClassName, 'Entity\\')->getFullName())
         ) {
-            // Mercure is needed
-            if (!class_exists(MercureExtension::class)) {
-                throw new RuntimeCommandException('Please run "composer require symfony/mercure". It is needed to broadcast entities.');
-            }
-
             $description = $command->getDefinition()->getOption('broadcast')->getDescription();
             $question = new ConfirmationQuestion($description, false);
             $isBroadcast = $io->askQuestion($question);
+
+            // Mercure is needed
+            if ($isBroadcast && !class_exists(MercureExtension::class)) {
+                throw new RuntimeCommandException('Please run "composer require symfony/mercure". It is needed to broadcast entities.');
+            }
 
             $input->setOption('broadcast', $isBroadcast);
         }

--- a/tests/Maker/MakeEntityTest.php
+++ b/tests/Maker/MakeEntityTest.php
@@ -600,6 +600,22 @@ class MakeEntityTest extends MakerTestCase
                 $this->runEntityTest($runner);
             }),
         ];
+
+        yield 'it_generates_entity_with_turbo_without_mercure' => [$this->createMakeEntityTest()
+            ->preRun(function (MakerTestRunner $runner) {
+                $runner->runProcess('composer require symfony/ux-turbo');
+            })
+            ->addExtraDependencies('twig')
+            ->run(function (MakerTestRunner $runner) {
+                $runner->runMaker([
+                    'User', // entity class
+                    'n', // no broadcast
+                    '',
+                ]);
+
+                $this->assertFileExists($runner->getPath('src/Entity/User.php'));
+            }),
+        ];
     }
 
     private function runEntityTest(MakerTestRunner $runner, array $data = []): void

